flake.nix

```diff
@@ -0,0 +1,54 @@
+{
+  description =
+    "eclair-lang: An experimental and minimal Datalog that compiles to LLVM";
+  inputs = {
+    np.url = "github:nixos/nixpkgs?ref=haskell-updates";
```

Comment:
Pin this to current commit of haskell-updates branch to make the build deterministic over time

Reply:
it is pinned, see flake.lock